crates/telemetry/src/env.rs

```diff
@@ -1,18 +1,33 @@
-/// Returns a boolean indicating if the OTEL layer should be enabled.
+/// Returns a boolean indicating if the OTEL tracing layer should be enabled.
 ///
 /// It is considered enabled if any of the following environment variables are set and not empty:
 /// - `OTEL_EXPORTER_OTLP_ENDPOINT`
 /// - `OTEL_EXPORTER_OTLP_TRACES_ENDPOINT`
-/// - `OTEL_EXPORTER_OTLP_METRICS_ENDPOINT`
 ///
 /// Note that this is overridden if OTEL_SDK_DISABLED is set and not empty.
-pub(crate) fn otel_enabled() -> bool {
-    const ENABLING_VARS: &[&str] = &[
+pub(crate) fn otel_tracing_enabled() -> bool {
+    otel_enabled(&[
         "OTEL_EXPORTER_OTLP_ENDPOINT",
```

Comment:
Somewhat of a tangent: these are (all?) available as consts from `opentelemetry-otlp`, e.g. https://docs.rs/opentelemetry-otlp/0.15.0/opentelemetry_otlp/constant.OTEL_EXPORTER_OTLP_ENDPOINT.html

It can be nice to use constants even when they are strings that match the name exactly; it both prevents typos and acts as implicit documentation that this is a standard.